README.md

```diff
@@ -32,38 +32,41 @@ cd lsk-faucet
 ```
 
 2. Bundle Frontend web with Vite
+
+**NOTE**: Please make sure to replace token icon at `web/public/token.png` with the specific ERC20 token icon.
```

Comment:
Would it make sense to replace the note with something like below? The following is just a format recommendation where we must list all the supported image formats and recommend the image resolution.

```suggestion
**NOTE**: Please make sure to update the token icon under `web/public/` with the specific ERC20 token icon. The file must be named either `token.png` or `token.jpg`. We recommend the image dimensions to be 50px x 50px.
```